### ISSUE TITLE
Manage name replacements in filters

### ISSUE THREAD

**Issue by hllwd** [briocheio/localFront#1]

**Comment by hllwd** [comment]
Currently, *lF* is able to manage a simple url as a filter. There are 2 options : 

* manage filename replacement in filters with an array of filename replacement in the filter
* allow the user to reorder the filters ; as we selected the first filter found, the reordering can provide a solution

**Comment by hllwd** [comment]
A simple string field will be sufficient for this milestone, with an edit field more sophisticated, allowing to enter multiple associations string1=>string1Replacement

```javascript
replacements: 'string1=>replacement1 string2=>replacement2'
```

**Comment by hllwd** [comment]
something like this https://github.com/briocheio/localFront/tree/issue/1-name-replacement

**Comment by hllwd** [comment]
merged in v0.1.4
